core/domain/suggestion_registry.py

```diff
@@ -906,6 +906,12 @@ def pre_update_validate(self, change):
                 'The new change skill_id must be equal to %s' %
                 self.change.skill_id)
 
+        if (self.change.skill_difficulty == change.skill_difficulty) and (
+                self.change.question_dict == change.question_dict):
+            raise utils.ValidationError(
+                'The new change skill_difficulty or question_dict should be'
```

Comment:
"At least one of the new skill_difficulty or question_dict should be changed."

Reply:
Done